code/modules/mob/living/carbon/xenomorph/castes/runner/runner.dm

```diff
@@ -52,3 +52,47 @@
 	holder.pixel_x = 24
 	holder.pixel_y = 24
 	hud_list[XENO_EVASION_HUD] = holder
+
+/mob/living/carbon/xenomorph/runner/MouseDrop(atom/over, src_location, over_location, src_control, over_control, params)
+	. = ..()
+	if(!ishuman(over))
+		return
+	if(!back)
+		balloon_alert(over,"This runner isn't wearing a saddle!")
+		return
+	if(!do_after(over, 3 SECONDS, NONE, src))
+		return
+	var/obj/item/storage/backpack/marine/duffelbag/xenosaddle/saddle = back
+	dropItemToGround(saddle,TRUE)
+
+/mob/living/carbon/xenomorph/runner/grabbed_self_attack()
+	if(!ishuman(pulling))
+		return NONE
+	var/mob/living/carbon/human = pulling
+	if(human.stat != DEAD)
+		//so aslong as the human isnt dead, put them on our back
+		INVOKE_ASYNC(src, PROC_REF(carry_human), human)
+		return COMSIG_GRAB_SUCCESSFUL_SELF_ATTACK
+	return NONE
+
+/mob/living/carbon/xenomorph/runner/proc/carry_human(mob/living/carbon/target, forced = FALSE)
+	if(!back)//cant ride without a saddle
```

Comment:
Check specifically for the saddle type rather than if there is anything on the back at all (`istype` is safe with null values)